src/test/spec/initial_dns_seedlist_discovery.rs

```diff
@@ -136,37 +137,19 @@ async fn run_test(mut test_file: TestFile) {
             "skipping initial_dns_seedlist_discovery test case due to TLS requirement mismatch",
         )
     } else {
-        // If the connection URI provides authentication information, manually create the user
```

Comment:
previously, we unconditionally ran a `ping`, which the spec didn't strictly require, and doing so required creating the user in those cases.
the spec does now require a ping, but only for tests with `ping`: true. the ones with `ping` set to true are ones without user information provided, so this code is no longer needed.